### PR TITLE
Add preliminary docs on aws

### PR DESCRIPTION
**Pull request by kasper-piskorski**
# Why is this PR needed?

# What does the PR do?

# Does it break backwards compatibility?

# List of future improvements not on this PR

### CODE REVIEW

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+TBC
+
+### Stack parameters
+The following parameters allow to define the stack details:
+
+* General:
+    - **StackName**
+    
+        Name for you Grakn KGMS stack. This must be a valid system name, specifically, it must consist of only lowercase letters, numbers, and hyphens and cannot exceed 50 characters.
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: *The* name for *your* Grakn KGMS stack...

Comment (lolski) [reply docs/pages/docs/01-cloud-deployment/aws-deployment.md]: ...and hyphens*,* and cannot exceed*s* 50 characters.

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        
+        The key pair assigned to Grakn instances to allow SSH access.
+    
+* Network:
+    - **VPC**
+    
+        The id of the VPC the stack should be deployed into.
+    - **InternetFacingRouteTable**
+      
+        The id of a internet-facing Route Table in the VPC.
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: The id of a*n* internet-facing...

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        The IP address range that can be used to access Grakn instances using SSH.
+    - **gRPCLocation**
+   
+        The IP address range that can be used to make gRPC requests.
+     
+
+* Cluster:
+    - **GraknInstanceType**
+    
+        EC2 instance type of a single Grakn instance.
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: EC2 instance type which will host Grakn.

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    - **GraknGroupSize**
+    
+        Number of Grakn instances in the cluster.
+    - **EbsVolumeSize**
+    
+        Size in GB of each of EBS volumes that are attached to Grakn instances.
+    - **OptimiseForEbs**
+    
+        Specifies whether the Grakn launch configuration should optimised for EBS I/O.
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: Specifies whether the Grakn launch configuration should *be* optimised for EBS I/O.

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+        
+## Accessing Grakn
+
+There are various ways to access Grakn on AWS. Here we will address the most common usage patterns.
+
+### Using gRPC
+
+The most common access pattern is to use a gRPC client to connect to a Grakn cluster. The connection happens via TCP port 48555 and the `gRPCLocation` stack parameter defines
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: ### Using [Grakn gRPC client](http://dev.grakn.ai/docs/java-library/setup)
The most common access pattern is to use the *[Grakn gRPC client](http://dev.grakn.ai/docs/java-library/setup)* to connect to a Grakn cluster

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+the range of IP addresses that can schedule gRPC requests.
+        
+### Logging into a node
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: ### Logging *in* into a node

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+To log into the Graql console, simply type `graql console`. After entering the user credentials (user: grakn, password: the one from Google console), you are free to interact with Grakn via the Graql terminal. 
+A succesful login attempt shall look like this:
+
+![](/images/aws-graql-console.png)
+
+A summary of available commands can be found [here](http://dev.grakn.ai/docs/get-started/graql-console).
+
+#### Accessing the Grakn console
+The Grakn console can be accessed similarly to graql console by typing:
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: ...similarly to *G*raql console by typing:

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+     
+A sample output of the command executed on a node of a healthy 3-node cluster shall look similar to this:
+
+![](/images/aws-cluster-health.png)
+
+#### Accessing the Graql console
+
+To access the Graql console, a user password is required. The default user password is specified in the `GraknUserPassword` output.
+
+To log into the Graql console, simply type `graql console`. After entering the user credentials (user: grakn, password: the one from Google console), you are free to interact with Grakn via the Graql terminal.
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: perhaps:

...(user: grakn, password: the one specified in the `GraknUserPassword` output),...

is more accurate?

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+![](/images/aws-graql-console.png)
+
+A summary of available commands can be found [here](http://dev.grakn.ai/docs/get-started/graql-console).
+
+#### Accessing the Grakn console
+The Grakn console can be accessed similarly to graql console by typing:
+
+`grakn console start`
+  
+and providing the user credentials (user: grakn, password: the one from Google console). A successful login attempt will look like this:
```
Comment (lolski) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: perhaps:

...(user: grakn, password: the one specified in the `GraknUserPassword` output),...

is more accurate?

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+#### Accessing the Graql console
+
+To access the Graql console, a user password is required. The default user password is specified in the `GraknUserPassword` output.
+
+To log into the Graql console, simply type `graql console`. After entering the user credentials (user: grakn, password: the one from Google console), you are free to interact with Grakn via the Graql terminal. 
+A succesful login attempt shall look like this:
+
+![](/images/aws-graql-console.png)
+
+A summary of available commands can be found [here](http://dev.grakn.ai/docs/get-started/graql-console).
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: "A summary of _the_ available commands"

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md (hunk 7 above)
Comment (jmsfltchr) [reply docs/pages/docs/01-cloud-deployment/aws-deployment.md]: ...similarly to _the_ Graql console by typing:

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+#### Accessing the Grakn console
+The Grakn console can be accessed similarly to graql console by typing:
+
+`grakn console start`
+  
+and providing the user credentials (user: grakn, password: the one from Google console). A successful login attempt will look like this:
+
+![](/images/aws-grakn-console.png)
+
+Provided you log in as user with `admin` privileges, Grakn console allows to perform the following actions:
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: "Grakn console allows _you_ to perform"

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+
+* create a new user:
+
+`CREATE USER username WITH PASSWORD userpassword WITH ROLE admin`
+
+* update existing user's password
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: update _an_ existing user's password

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+
+`UPDATE username WITH PASSWORD newpassword`
+
+* retrieve all the users present:
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: retrieve all _of_ the users present:

File: docs/pages/docs/01-cloud-deployment/best-practices.md [hunk 14]
```diff
@@ -8,27 +8,52 @@ permalink: /docs/cloud-deployment/best-practices
 folder: docs
 ---
 
-# GC Deployment Best Pracices
+# GC Deployment Best Practices
 
 In this section we shall describe the recommendations for comput and storage aspects of GC deployments
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/best-practices.md]: the recommendations for _computational/compute_ and storage aspects

Is compute the right name? Don't know what Google uses

File: docs/pages/docs/01-cloud-deployment/best-practices.md [hunk 15]
```diff
@@ -8,27 +8,52 @@ permalink: /docs/cloud-deployment/best-practices
...
 
 ## Compute
 
 The optimum machine choice offering a good balance between CPU and memory should be equipped with at least 4 vCPUs and 8 GB of RAM.
 Using machines with extra RAM amount above a 25 GB threshold is not expected to yield significant performance improvements.
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/best-practices.md]: Using machines with _additional RAM beyond a 25 GB threshold_ is not expected to yield significant performance improvements.

File: docs/pages/docs/01-cloud-deployment/best-practices.md [hunk 16]
```diff
@@ -8,27 +8,52 @@ permalink: /docs/cloud-deployment/best-practices
...
-Having these bounds in mind the following machines are recommended because they offer a balancd system resources for a range of workloads:
+Having these bounds in mind the following machines are recommended because they offer a balanced system resources for a range of workloads:
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/best-practices.md]: balanced _set of_ system resources for a

File: docs/pages/docs/01-cloud-deployment/gc-deployment.md [hunk 17]
```diff
@@ -70,7 +70,7 @@ The Grakn console can be accessed similarly to graql console by typing `grakn co
 
 ![](/images/gc-grakn-console.png)
 
-Provided you log in as user with`admin` privileges, Grakn console allows to perform the following actions:
+Provided you log in as user with `admin` privileges, Grakn console allows to perform the following actions:
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/gc-deployment.md]: Grakn console allows _you_ to perform

File: docs/pages/docs/01-cloud-deployment/overview.md [hunk 18]
```diff
@@ -0,0 +1,42 @@
...
+you can easily start up a familiar software package without having to manually configure the software, virtual machine instances, storage, or network settings.
+ 
+Deploying Grakn on Google Cloud is a straight-forward process, for a guide on how to get started, please visit: 
+
+[GC Deployment](https://dev.grakn.ai/docs/cloud-deployment/gc-deployment)
+
+
+## Amazon Web Services
+
+Amazon Web Services provide on-demand cloud computing platforms on a paid subscription basis. The technology allows subscribers to have at their disposal a full-fledged virtual cluster of computers,
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/overview.md]: _fully-fledged_ virtual cluster

**Comment by kasper-piskorski** [comment]
@lolski @jmsfltchr re-review please

File: docs/pages/docs/01-cloud-deployment/aws-deployment.md [hunk 19]
```diff
@@ -47,7 +47,7 @@ The following parameters allow to define the stack details:
 * Cluster:
     - **GraknInstanceType**
     
-        EC2 instance type of a single Grakn instance.
+        EC2 instance type of a Grakn instances.
```
Comment (jmsfltchr) [inline docs/pages/docs/01-cloud-deployment/aws-deployment.md]: EC2 instance type of Grakn instances.

**Pull request merged**